.github/ISSUE_TEMPLATE/feature_work_template.md

```diff
@@ -18,20 +18,12 @@ _As a [type of user], I want to [do something] [to produce this outcome]_
 -----------------------------------------
 
 ### Tasks
-- [ ] _Key pieces of work that will need to be done to meet the User Story and CoAs_
+#### UX
+
+#### Dev
 
 ### Definition of Done Checklist
 - [ ] Usability validated
```

Comment:
wonder if it would be helpful to identify who is responsible to check off the items?
- [ ] Usability validated (UX Team)